client/src/components/_utils/useToggle.js

```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const useToggle = (initialValue = false) => {
```

Comment:
I added a custom toggle hook for us to use. It means we can just call the `set_____________` to toggle the state without passing any arguments like (`!aThing`).